user_manual/changes/T701-019.yaml

```diff
@@ -0,0 +1,7 @@
+type: bugfix
+title: Fix ``zip_with_params`` on incomplete type decls
+description: |
+    This fixes an issue with calling ``P_Zip_With_Params`` on an discriminant
+    constraint that constrains the incomplete view of a discriminated type
+    would yield an empty result.
+title: 02-07-2020
```

Comment:
Pasto (`title` should be `date`), and the date format is wrong (`2020-07-02`)

Reply:
Thanks! I pushed too soon (testsuite wasn't done running :grimacing: ) Fixed!